plugins/kotlin/j2k/shared/tests/testData/newJ2k/annotations/annotationUsages.k1.kt

```diff
@@ -18,7 +18,8 @@ import javaApi.E
 @Anon5(1)
 @Anon6("x", "y")
 @Anon7(
-    String::class, StringBuilder::class
+    String::class,
```

Comment:
Formatting changes are because we now run test J2K inside of a command? Does this indeed reflect production J2K results?

Reply:
So it seems! I have no idea why, but it lines up with the results i get if I try converting a similar expression in a test instance of the IDE